src/metabase/api/public.clj

```diff
@@ -113,15 +123,7 @@
     (m/mapply card-api/run-query-for-card-async card-id export-format
```

Comment:
do we need this `m/apply` here?

Reply:
yeah, if you look down a couple of lines we pass an `options` map as the last parameter